.gitignore

```diff
@@ -3,3 +3,9 @@ Brewfile.lock.json
 
 # Crystal shards
 /lib/
+
+# Feature flags
+/feature-flags/
+
+# mitmproxy config
+/mitmproxy.yml
```

Comment:
I'm gitignoring this because it reveals some domains that I access. While that's not a big deal, I'd rather keep that list private.